CMakeLists.txt

```diff
@@ -487,6 +487,9 @@ set(tests_root ${CMAKE_CURRENT_BINARY_DIR}/src/tests)
 
 
 if(opencoarrays_aware_compiler)
+  if (RUN_DEVELOPER_TESTS)
+    message ( STATUS "Running Developer tests is enabled." )
+  endif()
```

Comment:
Note that there are cases where the developer tests will run, even if this is false... I'll adjust the output and the conditional to be clearer